client/components/AddBank.jsx

```diff
@@ -22,5 +22,5 @@ const AddBank = ({ userId, dispatch }) => {
 };
 
 export default connect(state => ({
-  userId: state.user.userId,
+  id: state.user.id,
```

Comment:
The id key is being destructured as-is in plaid.js line 25 to find a specific user whose primary key matches the id being passed down, and also passed into the bulkCreate accounts as an arg to provide the user_id you mentioned to associate an account to the user - user_id isn't a field in the users schema